packages/typescript-eslint/src/config-helper.ts

```diff
@@ -90,6 +90,11 @@ export function config(
     if (extendsArr == null || extendsArr.length === 0) {
       return config;
     }
+    if (!extendsArr.every(Boolean)) {
+      throw new Error(
+        'Some of your extensions are undefined, likely due to a problem with their import path.',
+      );
+    }
```

Comment:
This is accurate, but not very precise. If I have a half dozen or more plugins and only one is falsy then it'd be rather annoying to have to find out which exists with my own commenting / etc.

Proposal: how about having this mention which index or indices are falsy?